jwst/master_background/master_background_mos_step.py

```diff
@@ -85,6 +87,14 @@ def process(self, data):
             - "masterbkg_2d": `~jwst.datamodels.MultiSlitModel`
                 The 2D slit-based version of the master background.
 
+       sigma_clip : None or float
```

Comment:
Spacing is off by one here, causing a documentation error.
```suggestion
        sigma_clip : None or float
```